DEPS

```diff
@@ -1,26 +1,30 @@
 use_relative_paths = True
 
 deps = {
-  "vendor/requests": "https://github.com/kennethreitz/requests@e4d59bedfd3c7f4f254f4f5d036587bcd8152458",
+  "components/playlist/browser/resources/playlist-component": {
+    "url": "https://github.com/brave/playlist-component.git@5434730bf7342f1ba5c057f1640882bb38604a85",
+    "condition": "not checkout_ios",
+  },
+  "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
```

Comment:
why are these all moved around? It makes it hard to tell if something was changed or not

Reply:
Was suggested by @goodov from https://github.com/brave/brave-core/pull/16901#discussion_r1090275975